Encryptors/HaliteEncryptor.php

```diff
@@ -40,7 +40,14 @@ public function encrypt($data)
      */
     public function decrypt($data)
     {
-        return \ParagonIE\Halite\Symmetric\Crypto::decrypt($data, $this->getKey());
+        $data = \ParagonIE\Halite\Symmetric\Crypto::decrypt($data, $this->getKey());
+
+        if (is_object($data) && $data instanceof HiddenString)
```

Comment:
you don't need the `is_object` check, if $data is not an object `instanceof` does not error but simply return false. https://3v4l.org/dPTKc

Reply:
Fixed, thanks :)